This patch mitigates a bug in glade-2, which silently deletes stock_ids, which 
it thinks are not valid.   You may have to apply this patch after editing 
data-editor.glade with glade-2.  Glade-3 is another story and might give 
problems of its own.
--- src/ui/gui/data-editor.glade,old	2006-12-31 07:29:39.000000000 +0900
+++ src/ui/gui/data-editor.glade	2006-12-31 07:30:04.000000000 +0900
@@ -653,6 +653,7 @@
 		  <property name="visible">True</property>
 		  <property name="sensitive">False</property>
 		  <property name="tooltip" translatable="yes">Variables</property>
+                  <property name="stock_id">pspp-goto-variable</property>
 		  <property name="label" translatable="yes"></property>
 		  <property name="use_underline">True</property>
 		  <property name="visible_horizontal">True</property>
@@ -712,6 +713,7 @@
 		  <property name="visible">True</property>
 		  <property name="sensitive">False</property>
 		  <property name="tooltip" translatable="yes">Insert Case</property>
+                  <property name="stock_id">pspp-insert-case</property>
 		  <property name="label" translatable="yes"></property>
 		  <property name="use_underline">True</property>
 		  <property name="visible_horizontal">True</property>
@@ -730,6 +732,7 @@
 		  <property name="visible">True</property>
 		  <property name="sensitive">False</property>
 		  <property name="tooltip" translatable="yes">Insert Variable</property>
+                  <property name="stock_id">pspp-insert-variable</property>
 		  <property name="label" translatable="yes"></property>
 		  <property name="use_underline">True</property>
 		  <property name="visible_horizontal">True</property>
@@ -760,6 +763,7 @@
 		  <property name="visible">True</property>
 		  <property name="sensitive">False</property>
 		  <property name="tooltip" translatable="yes">Split File</property>
+                  <property name="stock_id">pspp-split-file</property>
 		  <property name="label" translatable="yes"></property>
 		  <property name="use_underline">True</property>
 		  <property name="visible_horizontal">True</property>
@@ -777,6 +781,7 @@
 		  <property name="visible">True</property>
 		  <property name="sensitive">False</property>
 		  <property name="tooltip" translatable="yes">Weight Cases</property>
+                  <property name="stock_id">pspp-weight-cases</property>
 		  <property name="label" translatable="yes"></property>
 		  <property name="use_underline">True</property>
 		  <property name="visible_horizontal">True</property>
@@ -794,6 +799,7 @@
 		  <property name="visible">True</property>
 		  <property name="sensitive">False</property>
 		  <property name="tooltip" translatable="yes">Select Cases</property>
+                  <property name="stock_id">pspp-select-cases</property>
 		  <property name="label" translatable="yes"></property>
 		  <property name="use_underline">True</property>
 		  <property name="visible_horizontal">True</property>
@@ -823,6 +829,7 @@
 		<widget class="GtkToggleToolButton" id="togglebutton-value-labels">
 		  <property name="visible">True</property>
 		  <property name="tooltip" translatable="yes">Value Labels</property>
+                  <property name="stock_id">pspp-value-labels</property>
 		  <property name="label" translatable="yes"></property>
 		  <property name="use_underline">True</property>
 		  <property name="visible_horizontal">True</property>
